onnxruntime/test/providers/cpu/nn/conv_transpose_op_test.cc

```diff
@@ -30,7 +30,7 @@ void TestConvTransposeOpInitializer(const ConvTransposeOpAttributes& attributes,
                                     bool is_filter_initializer = false,
                                     OpTester::ExpectResult expect_result = OpTester::ExpectResult::kExpectSuccess,
                                     const std::string& err_str = "",
-                                    const std::unordered_set<std::string>& excluded_provider_types = {kTensorrtExecutionProvider}) {
+                                    const std::unordered_set<std::string>& excluded_provider_types = {kTensorrtExecutionProvider, kOpenVINOExecutionProvider}) {
```

Comment:
but you've overrided this in all the TestConvTransposeOp() calls below?
you should undo all your modifications there.

Reply:
Got it George. I will revert the last commit.
I see, there are 2 failing convtranspose test with TensorRT, I will disable these 2 specific tests for TensorRT.

2021-08-30T19:46:08.6933821Z 1: [  FAILED  ] 2 tests, listed below:
2021-08-30T19:46:08.6934377Z 1: [  FAILED  ] **ConvTransposeTest.ConvTranspose_2D_OutputShape_1**
2021-08-30T19:46:08.6934994Z 1: [  FAILED  ] **ConvTransposeTest.ConvTranspose_2D_OutputShape_1_group_2_for_tranpose_path**